qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -130,13 +130,16 @@ def convert_acquire_deprecated(self, shift, instruction):
             dict: Dictionary of required parameters.
         """
         meas_level = self._run_config.get('meas_level', 2)
+        mem_slot = []
+        if instruction.mem_slot:
+            mem_slot = [instruction.mem_slot.index]
```

Comment:
why this if statement is not used for `register_slot`?

Reply:
It is, see line 155. The difference it seems is that `'memory_slot'` is always included in qobj, but `'register_slot'` is only included if it's not empty.

Reply:
If we assume `memory_slot` is always included, I think this should raise an error.

Reply:
I don't want to change any of the underlying behavior here. It seems that the backend requires `memory_slot` but doesn't mind if it's empty 🤷 Feel free to open an issue and we can continue discussion there about validation

Reply:
The backend will raise an issue if classical register is used. Either a memory slot or classical register must be present. So therefore if a memory slot is not present the backend will fail.